spec/requests/container_images_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+#
+# REST API Request Tests - /api/vms
+#
+describe "ContainerImages API" do
+  let(:provider) { FactoryGirl.create(:ems_kubernetes) }
+  let(:container_image) { FactoryGirl.create(:container_image, :ext_management_system => provider) }
+  let(:conatiner_id) { container_image.id }
+  let(:invalid_container_image) { container_image.id + 100 }
+  let(:invalid_image_url) { api_provider_container_image_url(nil, provider, invalid_container_image) }
+
+  context "ContainerImages show action" do
+    it "read a Container Image" do
+      api_basic_authorize(action_identifier(:container_images, :show, :subresource_actions, :get))
+      get api_provider_container_image_url(nil, provider, container_image)
+
+      expect(response).to have_http_status(:ok)
+      expect_single_resource_query("id" => container_image.id.to_s)
+    end
+  end
+
+  context "ContainerImages scan action" do
+    it "scans an invalid container_image" do
```

Comment:
Reading this description, it looks like you're saying it does the opposite of what's intended. Perhaps:

```ruby
it "responds 404 Not Found for an invalid container image"
```